src/guide/code-owners.md

```diff
@@ -53,58 +56,92 @@ This list is intended to make it easier to identify which email list to include
     * `libadt` – Compiler
     * `logging` – Runtime
     * `memory` – Runtime, GC
-    * `metaprogramming` – Runtime
-    * `oops` – Runtime
+    * `metaprogramming` – HotSpot
+    * `oops` – GC, Runtime
     * `opto` – Compiler
-    * `precompiled` – Runtime
+    * `precompiled` – HotSpot
     * `prims` – Runtime, Serviceability
     * `runtime` – Runtime
     * `services` – Runtime
-    * `shark` – Compiler
-    * `trace` – Runtime
-    * `utilities` – Runtime
+    * `utilities` – GC, Runtime
+  * `share/cds` – Runtime
 * `java.base`
-  * Core Libs should almost always be included but LangTools, HotSpot, Security and/or I18n may also be involved.
+  * Core Libs should almost always be included but Java Language, HotSpot, Security and/or I18n may also be involved.
   * `classes`
     * `crypto` – Security
-    * `internal` – HotSpot, Core Libs
+    * `internal`
+      * `access` – Core Libs, Security
+      * `event` – JFR
+      * `foreign` – Core Libs
+      * `icu` – Core Libs
+      * `io` – NIO
+      * `javac` – Java Language
+      * `jimage` – Client Libs
+      * `jmod` – HotSpot
+      * `jrtfs` – Core Libs
+      * `loader` – Core Libs
+      * `logger` – Core Libs
+      * `math` – Core Libs
+      * `misc` – Core Libs, HotSpot
+      * `module` – Core Libs
+      * `org/objectweb` – Core Libs
+      * `org/xml` – Core Libs
+      * `perf` – Runtime
+      * `platform` – HotSpot
+      * `ref` – Core Libs, GC
+      * `reflect` – Core Libs
+      * `util/random` – Core Libs
+      * `util/regex` – Core Libs
+      * `util/xml` – Core Libs
+      * `vm` – HotSpot
     * `invoke` – Core Libs
     * `io` – NIO
-    * `lang` – Core Libs
-    * `launcher` – LangTools
-    * `math` – Core Libs
+    * `java`
```

Comment:
Missing `security` and several other directories under `java`.

Reply:
All the missing directories are because there are many paths with the same leaf directory, e.g:
src/java.base/unix/classes/sun/security
src/java.base/macosx/classes/apple/security
src/java.base/windows/classes/sun/security
src/java.base/windows/lib/security
src/java.base/share/classes/sun/security
src/java.base/share/classes/java/security
src/java.base/share/classes/javax/security
src/java.base/share/classes/com/sun/security
src/java.base/share/lib/security
src/java.base/share/conf/security

All of these belongs to the same area and the directory list would become huge if we include them all. Again, maybe some `*` could be used to show that the single entry refer to them all..? Suggestions welcome :-)